test_vectordraw.py

```diff
@@ -183,6 +183,22 @@ def test_check_segment_angle(self):
         self.assertFails(vd.check_segment_angle(self.check(-45, 9, errmsg=custom_errmsg), vectors),
                          custom_errmsg)
 
+    def test_check_points_on_line(self):
+        errmsg = 'The line vec does not pass through the correct points.'
+        vectors = {'vec': self.vector(1, 1, 5, 5)}
+        self.assertPasses(vd.check_points_on_line(self.check([[1, 1], [5, 5]]), vectors))
+        self.assertPasses(vd.check_points_on_line(self.check([[1, 2], [5, 4]]), vectors))
+        self.assertPasses(vd.check_points_on_line(self.check([[3.7, 2.3], [2.3, 3.7]]), vectors))
```

Comment:
I used the same default tolerance as everything else (1.0 for distances, 2.0 for angles).  Since these are arbitrary units, it's completely unclear whether a distance of 1 is big or small.  I think it's OK fot the test – it should just verify it does what it is supposed to do.